include/config/windowswitcher-field.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef __LABWC_WINDOWSWITCHER_FIELD_H
+#define __LABWC_WINDOWSWITCHER_FIELD_H
+
+struct windowswitcher_field {
+	char *content;
+	int width;
+	struct wl_list link; /* struct rcxml.windowswitcher.fields */
+};
```

Comment:
We could use an enum instead of `char *content` here, e.g. parsing the string in `rcxml.c` and then just setting the enum, that way the OSD doesn't have to do the string comparisons every time it renders.

And we can also check the `content` value while parsing the config and simply reject invalid ones. Currently it seems the OSD would spam the log in case a value is invalid.

Reply:
Very good point. Done.